### DISCUSSION TITLE
Support of Multimedia in Assessment / Questions

### DISCUSSION THREAD

**Discussion by baljeetsingh** [Sunbird-Ed/Community#338] [General Discussion]
Hello All,

**No GIF**
As of today when we create a question/answer it support to it to be a text, image or audio
BUT
There is no option to upload a GIF or A VIDEO 
![image](https://user-images.githubusercontent.com/344965/210541066-3725fe02-a558-41e2-adcd-701ddf0df649.png)

**Complex topics**
As an educator I would like to have option to upload GIF or VIDEO to for better and visual understanding .
e.g Video or GIF of skeleton to show the movement of joints

**Is This Already Done ?**
Is this something can be made enable via any configuration so gif upload and video upload is supported ?

**Comment by Krishnaj20** [comment]
@baljeetsingh - which editor are you using to create this content?

@surendrasinghs @vinukumar-vs @pallakartheekreddy

**Reply by baljeetsingh** [reply]
1. SUNBIRD ED
2. Login as content creator
3. Go to Workspace
4. Create Resource
5. Go to Create Question
6. Create Question Screen there is option to upload image
7. click on upload image and use - try selecting a GIF
8. you should see about message

**Reply by Krishnaj20** [reply]
@baljeetsingh - are you referring to this editor? https://knowlg.sunbird.org/learn/product-and-developer-guide/editors/editor

**Reply by baljeetsingh** [reply]
@Krishnaj20 yes

**Reply by Krishnaj20** [reply]
Since this is an editor related question please post it in [Knowlg](https://github.com/orgs/Sunbird-knowlg/discussions) forums.

**Reply by vinukumar-vs** [reply]
@baljeetsingh The .gif format is not supported by file-upload-editor. Which requires code changes to support any new formats. 

Present supported formats:
https://github.com/project-sunbird/sunbird-content-plugins/blob/release-5.1.0/org.ekstep.uploadcontent-1.5/editor/uploadapp.js#L48

**Reply by vinukumar-vs** [reply]
@baljeetsingh Video upload is already supporting by the editor. For .gif upload requires code changes as mentioned in the above comment.

Hope this answered your query. 
@Krishnaj20
